src/testing/euler_split_inviscid_taylor_green_vortex.cpp

```diff
@@ -22,26 +22,107 @@ TestsBase::TestsBase(parameters_input)
 // return quad_kin_energ;
 //}
 
+template<int dim, int nstate>
+double EulerTaylorGreen<dim, nstate>::compute_MK_energy(std::shared_ptr < DGBase<dim, double> > &dg, unsigned int poly_degree) const
+{
+ // Overintegrate the error to make sure there is not integration error in the error estimate
+// int overintegrate = 10 ;//10;
+ int overintegrate = 0 ;//10;
+ dealii::QGauss<dim> quad_extra(dg->max_degree+1+overintegrate);
+           //  dealii::FEValues<dim,dim> fe_values_extra(dealii::MappingQ<dim>(dg->max_degree+overintegrate), dg->fe_collection[poly_degree], quad_extra,
+           //  dealii::update_values | dealii::update_JxW_values | dealii::update_quadrature_points);
+            const dealii::Mapping<dim> &mapping = (*(dg->high_order_grid->mapping_fe_field));
+            dealii::FEValues<dim,dim> fe_values_extra(mapping, dg->fe_collection[poly_degree], quad_extra, 
+                    dealii::update_values | dealii::update_JxW_values | dealii::update_quadrature_points);
+// dealii::QGauss<dim> quad_extra(dg->fe_system.tensor_degree()+overintegrate);
+// dealii::FEValues<dim,dim> fe_values_extra(dg->mapping, dg->fe_system, quad_extra,
+//       dealii::update_values | dealii::update_JxW_values | dealii::update_quadrature_points);
+
+// const unsigned int n_quad_pts = fe_values_extra.n_quadrature_points;
+// std::vector<std::array<double,nstate>> soln_at_q(n_quad_pts);;
+
+ double total_kinetic_energy = 0;
+
+ // Integrate solution error and output error
+// typename dealii::DoFHandler<dim>::active_cell_iterator
+// cell = dg->dof_handler.begin_active(),
+// endc = dg->dof_handler.end();
+
+ std::vector<dealii::types::global_dof_index> dofs_indices (fe_values_extra.dofs_per_cell);
+
+             //const double gam = euler_physics_double.gam;
```

Comment:
Clean up commented code and indentation throughout this function. This test could ultimately be repackaged using the `flow_solver.cpp` as done in PR #119

Reply:
Fixed. I left it as it is as per issue #127